scripts/release

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Usage
+#
+# ./release
+#
+
+# publish
+# Clean up any leftover gems
+rm -f ./*.gem
+# Build the new gem
+gem build recurly.gemspec
+# Push what should be the only gem present
+gem push ./*.gem
```

Comment:
Noting that we could specify `gem push ./recurly-${RELEASE_TAG}.gem`, however, the glob is just fine given the rm.